drivers/media/i2c/ov5647.c

```diff
@@ -232,14 +601,71 @@ static struct ov5647_mode supported_modes_8bit[] = {
 			.width = 640,
 			.height = 480
 		},
-		ov5647_640x480,
-		ARRAY_SIZE(ov5647_640x480)
+		ov5647_640x480_8bit,
+		ARRAY_SIZE(ov5647_640x480_8bit)
 	},
-	/* more modes below here... */
```

Comment:
Why remove it here? it's still true that you could add more 8 bit modes here. The comment as a whole is pretty redundant though.